compiler/testData/codegen/box/ir/kt25405.kt

```diff
@@ -1,3 +1,5 @@
+// IGNORE_BACKEND_FIR: JVM_IR
+
```

Comment:
This is the only one regression, though. After having a correct metadata, proper serialization started, and found that, when serializing the extension receiver of `tableView` below, type parameter `T` for `Table<T>` didn't have a counter part, type argument in the converted IrType. This means that type arguments for generic types are missing in general. I left a TODO/comment about this on our doc.